packages/api-client/src/controllers/workspace.ts

```diff
@@ -109,4 +111,31 @@ export default class WorkspaceController {
 
     return await parseResponse<GlobalSearchResponse>(response)
   }
+
+  async getWorkspaceInvitations(
+    request: GetWorkspaceInvitationsRequest,
+    headers?: Record<string, string>
+  ): Promise<ClientResponse<GetWorkspaceInvitationsResponse>> {
+    const queryParams = new URLSearchParams({
+      page: request.page?.toString() || '0',
+      limit: request.limit?.toString() || '10',
+      sort: request.sort || '',
+      order: request.order || '',
+      search: request.search || ''
+    })
+
+    ;[...queryParams.keys()].forEach((key) => {
```

Comment:
Pagination parsing is already done by the `parsePaginationUrl` function. You can refer to line 85